deploy/monitor/operator_service.yaml

```diff
@@ -0,0 +1,17 @@
+# Monitor Service (Metrics)
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name:mobile-developer-console-operator
+  name: mobile-developer-console-operator
+spec:
+  ports:
+    - name: metrics
+      port: 8383
+      protocol: TCP
+      targetPort: 8383
+  selector:
+    name: mobile-developer-console-operator
+  sessionAffinity: None
+  type: ClusterIP
```

Comment:
this is not required. The operator SDK will create the service object for the metrics endpoint automatically.